api/edge/v1alpha1/types.go

```diff
@@ -20,6 +20,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+/******************** Start Placement Type *********************/
```

Comment:
Why do we need these start/end comments ?

Reply:
we do not. removed.